### DISCUSSION TITLE
Error when we opening at local host

### DISCUSSION THREAD

**Discussion by dj-mm** [Atri-Labs/atrilabs-engine#404] [Help: Installation & Start]
[](url)

**Comment by darshitac11** [comment]
Hi @dj-mm , can you please share screenshots so that we know more about your error?

**Reply by Shubhodeep100** [reply]
![Screenshot 2023-02-13 232526](https://user-images.githubusercontent.com/96099026/218650602-18936402-6e1d-4bc2-be01-891527c809df.jpg)
@darshitac11  I'm getting this error message on the localhost:4005 which I'm getting to visit on clicking the 'Build & Run' button after editing the HomePage of the Food delivery repo.
![Screenshot 2023-02-14 110244](https://user-images.githubusercontent.com/96099026/218651406-117681fd-d276-4f17-8440-49ec9e275687.jpg)
Here I'm getting the option to visit localhost:4005 to Build my app. But on the localhost:4005 tab, I'm getting the error message.
![Screenshot 2023-02-14 110846](https://user-images.githubusercontent.com/96099026/218651915-c5ddbbe7-3639-426e-b372-9ff0dcb369c6.jpg)
And on the same time, I'm getting this 'Traceback (most recent call last): ' error on my VSCODE-terminal.
Please help me fix this.
